.travis/install.sh

```diff
@@ -50,3 +50,15 @@ pip install -r requirements.d/development.txt
 pip install codecov
 python setup.py --version
 pip install -e .[fuse]
+
+if [ -d "$HOME/.rustup/toolchains" ] && [ -f "$HOME/.cargo/env" ]; then
+    source ~/.cargo/env
+    rustup toolchain install stable # also updates
+    rustup default stable
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source ~/.cargo/env
+fi
+
+cd test-wrapper
+cargo build --release
```

Comment:
Oh, I thought you were saying we should separate the lines so `install.sh` could be used where `test-wrapper` could not be built. You're saying separate them so we could build `test-wrapper` where `install.sh` is not used, correct? That makes a lot more sense. I'll separate them.